src/input-capture.c

```diff
@@ -52,7 +52,7 @@ static InputCapture *input_capture;
 
 static GQuark quark_request_session;
 
-GType input_capture_get_type (void);
+GType input_capture_get_type (void) G_GNUC_CONST;
```

Comment:
Unrelated, should be a separate commit please, same with the clipboard hunks here which are a separate thing too.

Reply:
Just to make sure i have this correct  these are the two commits you want ? 
1. commit that adds V2 methods 
2. commit that adds the clipboard chunks

Reply:
I think the point was to make the change about `G_GNUC_CONST` its own commit, since it's unrelated to everything else.